config/environments/development.rb

```diff
@@ -38,13 +38,16 @@
   # For wordpress template
   config.x.wordpress_template = {
     division_urls: { # get division from request url
-      '/' => :us,
+      'theworkingworld.test' => :us,
+      'labase.test' => :argentina
```

Comment:
Does this (or somewhere—maybe readme) deserve a comment about editing hosts file?

Reply:
There's a lot of ways to accomplish this that vary by platform. It might be worth mentioning something in the dev setup docs somewhere but I don't want us duplicating a lot of info on setting this sort of thing up. 

In the meantime, it defaults to the US division if none of them match, so you can still get a working test environment for the features other than division detection without that.